packages/catlog/src/simulate/ode/cclfo.rs

```diff
@@ -0,0 +1,33 @@
+//! Constant-coefficient linear first-order (CCLFO) differential equations.
```

Comment:
While I appreciate the specificity, the acronym "CCFLO" is non-standard and so I think we should avoid using it as our primary name throughout the API. I will suggest "linear ODE", which doesn't imply that the coefficients are constant but should be clear enough to anyone with applied math background.